### DISCUSSION TITLE
Possible typo $row Did you mean $errors?

### DISCUSSION THREAD

**Discussion by AKozlovsky** [rappasoft/laravel-livewire-tables#276] [Q&A]
Hello, I'm beginner with this package. I would like to ask how to do when I want to see a table but it returns a some error (see a picture below). Thank you for any help.

![image](https://user-images.githubusercontent.com/18039919/117553155-20da9400-b050-11eb-8759-6b5383a83339.png)
![image](https://user-images.githubusercontent.com/18039919/117553163-3354cd80-b050-11eb-888b-b64c9ea6bd1a.png)
![image](https://user-images.githubusercontent.com/18039919/117553171-3b147200-b050-11eb-896a-76c8a73ad0a3.png)

**Comment by rappasoft** [comment]
Did you publish and change any views? Also, it looks like you're looking for livewire/search/view but your actual view is livewire/search

**Reply by AKozlovsky** [reply]
So, before I published assets and changed views, so it returns a same error.

![image](https://user-images.githubusercontent.com/18039919/117579071-a2363300-b0f1-11eb-8623-a21dac52c7a8.png)
![image](https://user-images.githubusercontent.com/18039919/117579086-aeba8b80-b0f1-11eb-98b7-5c1306030221.png)
![image](https://user-images.githubusercontent.com/18039919/117579101-bb3ee400-b0f1-11eb-8d76-e724ae9d559a.png)

Please, advise me on exactly how to make when creating a table. Thank you.

**Reply by rappasoft** [reply]
You do not have a badges.success component. That is not part of the package, it's just an example. You should read the Laravel documentation in full.

**Reply by AKozlovsky** [reply]
When trying to display a table,it still doesn't show me anything on the page. So I don't know what mistakes I made.

![image](https://user-images.githubusercontent.com/18039919/117586123-63ff3a80-b116-11eb-853e-eed91f34306e.png)
![image](https://user-images.githubusercontent.com/18039919/117586130-70839300-b116-11eb-854c-d91e4f764f05.png)
![image](https://user-images.githubusercontent.com/18039919/117586137-82fdcc80-b116-11eb-8d2d-534cee3d215d.png)

**Reply by AKozlovsky** [reply]
And when I made these stuffs then it is not working. I don't know how to display a table in search.blade.php.

![image](https://user-images.githubusercontent.com/18039919/117586536-0b7d6c80-b119-11eb-9de7-37125db52d32.png)

**Reply by AKozlovsky** [reply]
Yep, finally I found a solution. What a good thing :-)

![image](https://user-images.githubusercontent.com/18039919/117586954-aa0acd00-b11b-11eb-9370-7504e6c7b9bb.png)
![image](https://user-images.githubusercontent.com/18039919/117586959-b000ae00-b11b-11eb-852e-d0ccac78b751.png)
